### ISSUE TITLE
[SCREENREADER]: Form review card edit button

### ISSUE THREAD

**Issue by Mottie** [department-of-veterans-affairs/va.gov-team#4701]
## Description

This issue was originally brought up with a Higher-Level Review a11y review ([ref](https://github.com/department-of-veterans-affairs/va.gov-team/issues/3566#issuecomment-559214317)), but is an issue associated with all forms. cc @jenstrickland

The problem is with the `ReviewCardField` component (at least two copies exist in the eBenefits & 526 all-claims forms) improperly handling focus<sup>1</sup>. After activating the review card edit button, focus is lost. The user gets no indication that inputs are available for editing, or that they need to press <kbd>Tab</kbd> to get to the first focusable element inside the review card.

\[1] Similar to issue #4613

## Acceptance Criteria

- [ ] The screenreader should read the "Edit" button as a button.
- [ ] The button should include wording to let the user know that they can interact with the button to enter an edit mode.
- [ ] After entering edit mode, focus should be placed on the first focusable element inside the review card.

## Steps to Recreate

1. Log in as user 218, then go to a form that contains a phone & email or mailing address review card, e.g. https://staging.va.gov/disability/file-disability-claim-form-21-526ez/contact-information 
2. Use the keyboard to <kbd>Tab</kbd> to the first "Edit" button.
3. The screen reader will say "Edit Phone & email", but doesn't notify the user that the using the button will switch the form into edit mode.

    <img src="https://user-images.githubusercontent.com/57469/69751098-5c172a80-111c-11ea-862a-809de857af59.png" alt="collapsed review card showing edit button" />

3. Press <kbd>Enter</kbd> to switch to edit mode.
4. At this point, focus is lost, but still remains on the `<body>`, with no message stating that the new content had become editable or interactive.

    <img src="https://user-images.githubusercontent.com/57469/69751199-954f9a80-111c-11ea-8453-86303d845169.png" alt="Expanded review card without focus" />

5. Pressing <kbd>Tab</kbd> will move focus to the next focusable element within that content

## Possible Fixes (optional)

See issue #4613 for a code example that will move focus to the first focusable element.

**Comment by Mottie** [comment]
This has been fixed as part of #2138 in this PR: https://github.com/department-of-veterans-affairs/vets-website/pull/11847
